components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -452,4 +453,52 @@ - (void)testTitlesNumberOfLinesPassedToViewsAfterItemsAssigned {
   }
 }
 
+- (void)testItemForPointInsideItemView {
+  // Given
+  UITabBarItem *item1 = [[UITabBarItem alloc] initWithTitle:@"1" image:nil tag:0];
+  UITabBarItem *item2 = [[UITabBarItem alloc] initWithTitle:@"2" image:nil tag:0];
+  self.bottomNavBar.frame = CGRectMake(0, 0, 320, 56);
+
+  // When
+  self.bottomNavBar.items = @[ item1, item2 ];
+  [self.bottomNavBar layoutIfNeeded];
+  UIView *view1 = [self.bottomNavBar viewForItem:item1];
+  UIView *view2 = [self.bottomNavBar viewForItem:item2];
+  UITabBarItem *result1 = [self.bottomNavBar tabBarItemForPoint:view1.center];
+  UITabBarItem *result2 = [self.bottomNavBar tabBarItemForPoint:view2.center];
+
+  // Then
+  XCTAssertEqualObjects(result1, item1);
+  XCTAssertEqualObjects(result2, item2);
+}
+
+- (void)testItemForPointOutsideOfItemView {
```

Comment:
Please add a test where the tap is inside the bar, but outside any items. You may have to "fudge" a bit, but making a bar 200 points high should certainly ensure that the items don't extend to the bottom. By default, the item views will be no more than 56 points high.

Reply:
Done.